### DISCUSSION TITLE
Is pyg-lib still required to install?

### DISCUSSION THREAD

**Discussion by Ripper346** [pyg-team/pytorch_geometric#8159] [Q&A] [answered]
Hi, this morning I was trying to install the latest version of pytorch geometric with torch 2.1.0, but it failed to install `pyg_lib` from https://data.pyg.org/whl/torch-2.1.0+cpu.html. I saw that the package is present only until torch 2.0.1. My question is, is this package still required and do I have to install it from torch 2.0.1 wheel link? Or at the current stage torch geometric has incompatibilities with torch 2.1.0? I don't even know what the purpose of that library is. Thank you

**Comment by rusty1s** [comment, marked as answer]
You can install the nightly version of `pyg-lib`, which has support for PyTorch 2.1.0. It is also not a required dependency to run PyG.
